addons/website/static/src/scss/website.scss

```diff
@@ -1122,6 +1122,33 @@ table.table_desc tr td {
     }
 }
 
+// Navbar templates
+.o_navbar_showcase{
+    .navbar-brand{
+        margin: auto;
+    }
+    .navbar-nav{
+        margin-top: 10px;
+    }
+}
+
+.o_navbar_brand{
+    .container{
+        -webkit-flex-wrap: wrap !important;
```

Comment:
Don't need vendor prefixes rules, we add them automatically for the browsers we support.